Makefile

```diff
@@ -12,7 +10,10 @@ update:
 	pip install -r requirements.txt
 
 migrate:
```

Comment:
imo no, since that might cause the deploy script to start making new migrations in prod.